serialization/pickle.ksy

```diff
@@ -0,0 +1,443 @@
+meta:
+  id: pickle
```

Comment:
I think it would be a good idea to rename this spec to `python_pickle` - the name `pickle` on its own is not that clear if you don't already know how Python's serialization format is called.

Also I just noticed that when you compile this spec to Python with the default ksc options, it produces a file called `pickle.py`, which conflicts with Python's `pickle` module 😃

Reply:
Will do.